### PR TITLE
Api/copy

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**
### Scope
Make input field searchable (temporary)

### Work Done
 - Added new set of short temporary APIs locally
 - Created new MedicineBasicSearch component
 - Display all results from fetch() -> No filtering of medicines yet as that can be done easier through the correct APIs

### CODE REVIEW

File: src/components/MedicineBasicSearch/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import tablet from '../../../static/images/pill_24.png';
+import capsule from '../../../static/images/capsule_24.png';
+import syrup from '../../../static/images/syrup_24.png';
+import classes from './style.module.css';
+
+const callImage = dosage_form => {
```
Comment (schalkventer) [inline src/components/MedicineBasicSearch/index.js]: A more functional programming way would be to do the following:

```
const calcIcon = (dosageForm) => {
  switch (dosageForm) {
    case 'suspension': return syrup;
    case 'tablet': return tablet;
    case 'effervescent tablet': return tablet;
    case 'capsule': return capsule;
    default: return tablet;
  }
}

const medIcon = calcIcon(dosage_form);
```

File: src/components/SearchTool/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  fetchData = () => {
+    fetch('http://localhost:8000/api/search-lite.json')
+      .then(response => response.json())
+      .then(parsedJSON => {
+        this.setState({results: parsedJSON})
+      })
+      .catch(error => console.log('parsing failed:', error));
+  }
+
+  submitForm = event => {
```
Comment (schalkventer) [inline src/components/SearchTool/index.js]: ESLint might complain about this.

File: src/components/MedicineBasicSearch/index.js (hunk 1 above)
Comment (ZeeshaanMaudar) [reply src/components/MedicineBasicSearch/index.js]: Got it.

**Comment by ZeeshaanMaudar** [comment]
Stale branch / PR; no longer relevant.

**Pull request closed without merging**